src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonResource.kt

```diff
@@ -137,8 +137,10 @@ class PrisonResource(private val prisonService: PrisonService, private val addre
       ),
     ],
   )
-  fun getPrisonNames(): List<PrisonNameDto> {
-    return prisonService.getPrisonNames()
+  fun getPrisonNames(
+    @Parameter(description = "Active", example = "true", required = false) @RequestParam active: Boolean? = null,
```

Comment:
I want to leave the original contract /names as it was so i set this as null as if i chose Boolean? = false as default if active is not chose then it gets this value = false

Reply:
You can set a default of false and not have it as a null
`@Parameter(description = "Active", example = "true", required = false, default="false") @RequestParam active: Boolean = false`

Reply:
The 'description' of the parameter would have to be improved, e.g. explaining that if not set all the prisons are returned regardless of active or not and if set then only the active or inactive ones are returned.

Reply:
@Mjwillis If i aset active as false then it looks for non active prisons if we dont use a param as it is as default and other tests of other resources fails